[2024-10-16T03:00:10.064+0000] {processor.py:186} INFO - Started process (PID=32909) to work on /opt/airflow/dags/send_alert.py
[2024-10-16T03:00:10.064+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/send_alert.py for tasks to queue
[2024-10-16T03:00:10.065+0000] {logging_mixin.py:190} INFO - [2024-10-16T03:00:10.065+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/send_alert.py
[2024-10-16T03:00:10.069+0000] {logging_mixin.py:190} INFO - [2024-10-16T03:00:10.068+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/send_alert.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/send_alert.py", line 1, in <module>
    from alerts import send_rocket_chat_message
ModuleNotFoundError: No module named 'alerts'
[2024-10-16T03:00:10.069+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/send_alert.py
[2024-10-16T03:00:10.081+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/send_alert.py took 0.020 seconds
[2024-10-16T03:00:29.373+0000] {processor.py:186} INFO - Started process (PID=33212) to work on /opt/airflow/dags/send_alert.py
[2024-10-16T03:00:29.374+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/send_alert.py for tasks to queue
[2024-10-16T03:00:29.376+0000] {logging_mixin.py:190} INFO - [2024-10-16T03:00:29.375+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/send_alert.py
[2024-10-16T03:00:29.384+0000] {logging_mixin.py:190} INFO - [2024-10-16T03:00:29.382+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/send_alert.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/send_alert.py", line 1, in <module>
    from alerts import send_rocket_chat_message
ModuleNotFoundError: No module named 'alerts'
[2024-10-16T03:00:29.384+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/send_alert.py
[2024-10-16T03:00:29.402+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/send_alert.py took 0.036 seconds
[2024-10-16T03:00:32.417+0000] {processor.py:186} INFO - Started process (PID=33217) to work on /opt/airflow/dags/send_alert.py
[2024-10-16T03:00:32.418+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/send_alert.py for tasks to queue
[2024-10-16T03:00:32.419+0000] {logging_mixin.py:190} INFO - [2024-10-16T03:00:32.419+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/send_alert.py
[2024-10-16T03:00:32.439+0000] {logging_mixin.py:190} INFO - [2024-10-16T03:00:32.438+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/send_alert.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/send_alert.py", line 1, in <module>
    from alerts import send_rocket_chat_message
ModuleNotFoundError: No module named 'alerts'
[2024-10-16T03:00:32.440+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/send_alert.py
[2024-10-16T03:00:32.451+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/send_alert.py took 0.040 seconds
[2024-10-16T03:01:02.975+0000] {processor.py:186} INFO - Started process (PID=33523) to work on /opt/airflow/dags/send_alert.py
[2024-10-16T03:01:02.976+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/send_alert.py for tasks to queue
[2024-10-16T03:01:02.977+0000] {logging_mixin.py:190} INFO - [2024-10-16T03:01:02.977+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/send_alert.py
[2024-10-16T03:01:02.983+0000] {logging_mixin.py:190} INFO - [2024-10-16T03:01:02.981+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/send_alert.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/send_alert.py", line 1, in <module>
    from alerts import send_rocket_chat_message
ModuleNotFoundError: No module named 'alerts'
[2024-10-16T03:01:02.983+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/send_alert.py
[2024-10-16T03:01:02.997+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/send_alert.py took 0.028 seconds
